tests/functional/requirements.txt

```diff
@@ -1,2 +1,2 @@
 git+https://github.com/openstack-charmers/zaza-openstack-tests.git#egg=zaza.openstack
-git+https://github.com/openstack-charmers/zaza.git@libjuju-3.1#egg=zaza
+git+https://github.com/jneo8/zaza.git@libjuju-3.1-config-cred#egg=zaza
```

Comment:
We can directly use the `libjuju-3.1` branch from upstream since the changes for configuring credentials has been merged.